SportsHub/SportsHub.Core/Repository/IUnitOfWork.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SportsHub.Domain.Repository
+{
+    public interface IUnitOfWork : IDisposable
+    {
+        IUserRepository Users { get; }
```

Comment:
lets rename it UserRepository

Reply:
Create UOW - folder in the same level that Repository Folder and move IUnitOfWork in that folder